.buildkite/x-pack/pipeline.xpack.packetbeat.yml

```diff
@@ -26,46 +25,262 @@ env:
+        command: |
+          Set-Location -Path x-pack/packetbeat
+          mage build unitTest
+        key: "extended-win-11-unit-tests"
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_WIN_11}"
+          machineType: "${GCP_WIN_MACHINE_TYPE}"
+          disk_size: 100
+          disk_type: "pd-ssd"
+        artifact_paths:
+          - "x-pack/packetbeat/build/*.xml"
+          - "x-pack/packetbeat/build/*.json"
+        notify:
+          - github_commit_status:
+              context: "x-pack/packetbeat: Windows 11 Unit Tests"
+
+      - label: ":windows: Windows 2019 Unit Tests"
+        command: |
+          Set-Location -Path x-pack/packetbeat
+          mage build unitTest
+        key: "extended-win-2019-unit-tests"
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_WIN_2019}"
+          machineType: "${GCP_WIN_MACHINE_TYPE}"
+          disk_size: 100
+          disk_type: "pd-ssd"
+        artifact_paths:
+          - "x-pack/packetbeat/build/*.xml"
+          - "x-pack/packetbeat/build/*.json"
+        notify:
+          - github_commit_status:
+              context: "x-pack/packetbeat: Windows 2019 Unit Tests"
+
+      #  #skip: "see https://github.com/elastic/beats/issues/38142"
+      # - label: ":windows: Windows 10 System Tests"
+      #   key: "extended-win-10-system-tests"
+      #   command: ".buildkite/scripts/win_unit_tests.ps1 systemtest"
+      #   agents:
+      #     provider: "gcp"
+      #     image: "${IMAGE_WIN_10}"
+      #     machineType: "${GCP_WIN_MACHINE_TYPE}"
+      #     disk_size: 100
+      #     disk_type: "pd-ssd"
+      #   artifact_paths:
+      #     - "x-pack/packetbeat/build/*.xml"
+      #     - "x-pack/packetbeat/build/*.json"
+
+  - group: "x-pack/packetbeat Linux arm Extended Tests"
+    key: "x-pack-packetbeat-extended-linux-arm-tests"
+    if: build.env("BUILDKITE_PULL_REQUEST") == "false" || build.env("GITHUB_PR_LABELS") =~ /.*arm.*/
+    steps:
+      - label: ":linux: Ubuntu ARM Unit Tests"
+        key: "extended-arm64-unit-test"
+        command: "cd x-pack/packetbeat && mage build unitTest"
+        if: build.env("GITHUB_PR_LABELS") =~ /.*arm.*/
+        agents:
+          provider: "aws"
+          imagePrefix: "${IMAGE_UBUNTU_ARM_64}"
+          instanceType: "${AWS_ARM_INSTANCE_TYPE}"
+        artifact_paths:
+          - "x-pack/packetbeat/build/*.xml"
+          - "x-pack/packetbeat/build/*.json"
+        notify:
+          - github_commit_status:
+              context: "x-pack/packetbeat: Ubuntu ARM Unit Tests"
+
+  - group: "x-pack/packetbeat MacOS Extended Tests"
+    key: "x-pack-packetbeat-extended-macos-tests"
+    if: build.env("BUILDKITE_PULL_REQUEST") == "false" || build.env("GITHUB_PR_LABELS") =~ /.*macos.*/
+    steps:
+      - label: ":mac: MacOS Unit Tests"
+        key: "extended-macos-unit-tests"
+        command: ".buildkite/scripts/unit_tests.sh"
+        if: build.env("GITHUB_PR_LABELS") =~ /.*macos.*/
```

Comment:
we want to run these tests when the pipeline gets triggered by a merge commit and since the conditional in line 222 above covers for both cases, should we remove this conditional, as done in other migrated pipelines?

Reply:
yep, correct, thanks for catching that